SEPADebit/src/androidTest/java/com/braintreepayments/api/SEPADebitClientTest.java

```diff
@@ -0,0 +1,23 @@
+package com.braintreepayments.api;
+
+import android.content.Context;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+// TODO: replace with integration tests
```

Comment:
Did you want to resolve this prior to merging?

Reply:
This is for future PR work so all good 👍